deployment/GP03FLMB_Deploy.csv

```diff
@@ -89,21 +89,39 @@ SR17-10,,SR1811,,GP03FLMB-RIS01-00-SIOENG000,5,1,2017-07-14T22:49:00,2018-07-25T
+SR1811,,,,GP03FLMB-RIM01-02-CTDMOH069,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-CTDMOH-13655,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIM01-02-CTDMOH070,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-CTDMOH-13652,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIM01-02-CTDMOH071,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-CTDMOH-13448,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIS01-00-SIOENG000,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGCON-SIOCON-13282,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIS01-03-DOSTAD000,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-DOSTAD-00415,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIS01-04-PHSENF000,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-PHSENF-P0079,50.329537,-144.397928,,,4145,
+SR1811,,,,GP03FLMB-RIS01-05-FLORTD000,6,1,2018-07-24T23:01:00,,CGMGP-03FLMB-00006,,CGINS-FLORTD-01435,50.329537,-144.397928,,,4145,
+SKQ201920S,,,,GP03FLMB-RIM01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGCON-SIOCON-13280,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-ADCPSL007,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-ADCPSL-18260,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG060,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-10218,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG061,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-11644,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG062,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-12189,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG063,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-11640,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG064,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-11533,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG065,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-10260,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG066,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-12406,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG067,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-11605,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOG068,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOG-12310,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH069,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12388,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH070,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12618,,,,,,
+SKQ201920S,,,,GP03FLMB-RIM01-02-CTDMOH071,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-CTDMOH-10228,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGCON-SIOCON-13284,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-03-DOSTAD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-DOSTAD-00519,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-04-PHSENF000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-PHSENF-P0147,,,,,,
+SKQ201920S,,,,GP03FLMB-RIS01-05-FLORTD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMB-00007,,CGINS-FLORTD-01298,,,,,,

```

Comment:
Looks good for pre-deployment status

Reply:
Are the lat lon correct? Surveyed anchor locations are (decimal minutes):
HYPM-7 50 06.911 N 144 54.967 W
FLMA-7 50 01.362 N 144 21.647 W
FLMB-7 50 22.533 N 144 30.870 W